src/catchup/CatchupWork.cpp

```diff
@@ -53,9 +53,25 @@ CatchupWork::~CatchupWork()
 std::string
 CatchupWork::getStatus() const
 {
-    return fmt::format(
-        "Catching up to ledger {}: {}", mCatchupConfiguration.toLedger(),
-        mCurrentWork ? mCurrentWork->getStatus() : Work::getStatus());
+    std::string toLedger;
+    if (mCatchupConfiguration.toLedger() == CatchupConfiguration::CURRENT)
```

Comment:
@jonjove Do you think it's valuable to log the ledger number when catching up to current? We could just log "CURRENT" instead.

Reply:
It's a good question. I think the answer is yes, because it makes it very explicit. Otherwise you might look back at logs and wonder to what ledger was it actually catching up.